assets/css/frontend-forms.css

```
@@ -782,30 +782,305 @@ body .wpuf_sub_info .wpuf-expire {
+  width: 20px;
+  height: 20px;
+  margin: 0;
+  cursor: pointer;
+}
+body .wpuf-gateway-card .wpuf-payment-instruction {
+  margin-top: 12px;
+  padding-top: 12px;
+  border-top: 1px solid #f3f4f6;
+}
+body li:has(input[type="radio"]:checked) .wpuf-gateway-card {
+  border-color: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)